App/src/features/projects/ProjectsList.js

```diff
@@ -25,6 +25,8 @@ export default class ProjectsList extends Component {
         this.renderProject = this.renderProject.bind(this);
         this.renderHeader = this.renderHeader.bind(this);
         this.renderAerProjects = this.renderAerProjects.bind(this);
+        this.renderCommingProjects = this.renderCommingProjects.bind(this);
+        this.renderCurrentProjects = this.renderCurrentProjects.bind(this);
```

Comment:
No need to bind `renderCommingProjects` and `renderCurrentProjects`. There are probably more functions that shouldn't be bind here. Only functions that are passed to children components, such as `TouchableOpacity` should be binded so they don't loose there context